### DISCUSSION TITLE
Pull-Down Menu Issue with TSML UI Japanese

### DISCUSSION THREAD

**Discussion by Hiro-Tokyo** [code4recovery/12-step-meeting-list#1172] [General]
Hello Team!

I thought it happened today or a few days ago.
On Kansai Area Website and Japan Online Intergroup, 'Anywhere' and 'Any Type' pull-down menu do not work correctly.
These show only a few addresses and types. Actually these were working well but issues happened suddenly.
So any solutions?

Kansai Area Website
https://aa-kansai-hp.jimdofree.com/%E3%83%9F%E3%83%BC%E3%83%86%E3%82%A3%E3%83%B3%E3%82%B0%E5%A0%B4%E6%A1%88%E5%86%85/

Japan Online Intergroup Website
https://aajapan-online-intergroup.org/meetings/

<img width="382" alt="Any Type" src="https://github.com/code4recovery/12-step-meeting-list/assets/100449629/a3e9b679-3a38-4dd4-a79e-7dd92bd1dd2f">
<img width="351" alt="Anywhere" src="https://github.com/code4recovery/12-step-meeting-list/assets/100449629/18dcf2b1-09c8-4a5f-b8f5-9cfa002eaf24">

**Comment by joshreisner** [comment]
hi @Hiro-Tokyo thank you for posting here. 

the issue should be fixed now. a change we made on july 8 seems to have broken japanese language type URLs - hopefully it is working for you again. you may need to hit refresh in your browser to see the types menu again

https://github.com/code4recovery/tsml-ui/commit/db9f29839d4de8d2736edbcd4e1fffa6db8f9324

**Reply by Hiro-Tokyo** [reply]
Thank you so much Josh! I made sure it was fixed!
